### PR TITLE
MC-5731: pipeline: Move APT repo management to dedicated VM

### PR DESCRIPTION
**Pull request by lluiscampos**
The responsibility of the CI, from now on, is to copy the
to-be-published files into /incoming directory in the S3 bucket. The VM
takes it from there.

Signed-off-by: Lluis Campos <lluis.campos@northern.tech>

### CODE REVIEW

**Comment by lluiscampos** [comment]
Goes with https://github.com/mendersoftware/sre-tools/pull/154

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -158,156 +158,16 @@ test:acceptance:
...
-    - else
-    -   copy_if_non_existent "${CI_PROJECT_DIR}/output/mender-configure*.deb" pool/main/m/mender-configure/
-    - fi
-
-    # Upload to bucket
-    - aws s3 sync db s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/db
-    - aws s3 sync --acl public-read pool s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/pool
-    - aws s3 sync --acl public-read dists s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/dists
+    # Upload .changes and .deb files
+    - for file in $(find output/ -name *.changes -o -name *.deb); do
```
Comment (tranchitella) [inline .gitlab-ci.yml:168]: @lluiscampos please upload deb packages/buildinfo first, thenthen the changes file. Is there a particular reason why you are not uploading the buildinfo file?

**Review by tranchitella — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (lluiscampos) [reply .gitlab-ci.yml:168]: > please upload deb packages/buildinfo first, thenthen the changes file.

I can do that. As I am taking a lock before any copy I didn't think about any particular order

> Is there a particular reason why you are not uploading the buildinfo file?

No reason at all. I just realized that we don't do any use of them from the script in the VM, so decided to drop. I can add them if you want.

Comment (tranchitella) [reply .gitlab-ci.yml:168]: The script on the VM processes all the files listed in `changes`, if you don't mind, I would upload them all, leaving the `changes` file last.

**Comment by tranchitella** [comment]
@lluiscampos can we finalize and test this PR?

**Review by tranchitella — APPROVED** [review]

**Pull request merged**